resources/lang/en/events/misc/general.json

```
@@ -2486,8 +2494,9 @@
       "mountainous"
     ],
     "season": ["any"],
+    "tags": ["clan_wide"],
     "weight": 10,
-    "event_text": "Tracks don't lie, even if o_c_n may. One of them has been traipsing across the border. It remains to be seen if that cat is a confused apprentice or a deliberate provacation.",
+    "event_text": "Tracks don't lie, even if o_c_n may. One of them has been traipsing across the border. It remains to be seen if that cat is a confused apprentice or a deliberate provocation.",
```

```suggestion
    "event_text": "Tracks don't lie, even if o_c_n may. One of them has been traipsing across the border. It remains to be seen if that cat is a confused apprentice or a deliberate provocateur.",
```
Since the first part is indicating that we're discussing the actor not the action, provocateur is the right word.